notes/2023/2023-04-20.md

```diff
@@ -0,0 +1,38 @@
+# 2023-04-20 ESLint TSC Meeting Notes
+
+## Transcript
+
+[`2023-04-20-transcript.md`](2023-04-20-transcript.md)
+## Attending
+
+* Nicholas C. Zakas (@nzakas) - TSC
+* Milos Djermanovic (@mdjermanovic) - TSC
+
+@nzakas moderated, and @sam3k took notes.
+
+## Topics
+
+
+### [feat: Add suppress ignored file warnings RFC](https://github.com/eslint/rfcs/pull/90)
+
+**TSC Summary:** We have decided that the RFC should be changed so if the warnIgnored option of lintText() is not specified, it will default to the warnIgnored value for the FlatESLint instance.
```

Comment:
```suggestion
Originally the RFC proposed using the value of the warnIgnored as the default for the warnIgnored option of the `lintText()` method. This was changed in a feedback round to avoid a breaking change. However, now that we're only going to implement this in `FlatESLint`, can we go back to the original version?

**Resolution:** We have decided that the RFC should be changed so if the warnIgnored option of lintText() is not specified, it will default to the warnIgnored value for the FlatESLint instance.
```